lib/components/Editor/CustomExtensions/useCustomExtensions.js

```diff
@@ -85,6 +86,14 @@ const useCustomExtensions = ({
         },
       })
     );
+
+    customExtensions.push(
+      SpecialMentions.configure({
+        suggestion: {
+          items: createMentionSuggestions(mentions),
```

Comment:
Add the following line in line 82.
```jsx
const items =  createMentionSuggestions(mentions);
```

Pass it to both the `configure` methods.
```suggestion
          items
```